controllers/rbac/aggregation_test.go

```diff
@@ -32,7 +32,7 @@ var _ = Describe("Aggregation Tests", func() {
 
 	JustBeforeEach(func() {
 		By("init new Aggregation")
-		a = NewAggregation(k8sManager, "default")
+		a = NewAggregation(k8sManager.GetClient(), "default")
```

Comment:
NIT: Can you use a little bit longer variable name? One letter is nice, but might be too (short) vague, especially when it used multiple times (also in _controllers/rbac/aggregation.go_ ). 
How about `agg`? :)

Reply:
Done here (even though it's unrelated to this PR).
In `aggregation.go` though its the method receiver name, and using just the first letter there is best practice, because its meaning is obvious.